### DISCUSSION TITLE
Photo overview/list per map section (in web)

### DISCUSSION THREAD

**Discussion by mvbjrn** [immich-app/immich#5018] [General]
As a web user that likes to view photos via the map, I would like to have a list of all photos for my current map section so that I get an overview about all photos of that location and am able to select those. 

Currently, I can only go through all of them but do not have an overview with thumbnails.

This is working nicely in the iOS app, but the behavior in the web application appears to be quite different.

OT: Thanks for all your work! My wife and I really enjoy using Immich!

**Comment by bjrne** [comment]
I think it's more or less a duplicate of #4519, maybe also cast a vote there :)
